### PR TITLE
Jar name has changed to reflect concord version

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: ansible/roles/concord-demo/templates/concord.conf.j2 [hunk 1]
```diff
@@ -0,0 +1,9 @@
+description "concord"
+
+start on startup
+
+respawn
+
+env LD_LIBRARY_PATH=/usr/local/lib
+env MESOS_NATIVE_JAVA_LIBRARY=/usr/local/lib/libmesos.so
+exec java -DMESOS_NATIVE_JAVA_LIBRARY=/usr/local/lib/libmesos.so -cp /usr/local/share/concord/concord-scheduler-'{{ concord_version }}'.jar com.concord.scheduler.Scheduler -l 127.0.0.1:11211 -m zk://127.0.0.1:2181/mesos -k zk://127.0.0.1:2181/kafka -c zk://127.0.0.1:2181/concord -n concord
```
Comment (bigs) [inline ansible/roles/concord-demo/templates/concord.conf.j2]: newline pls

Comment (bigs) [inline ansible/roles/concord-demo/templates/concord.conf.j2]: remove single quotes

**Comment by bigs** [comment]
two baby changes, otherwise lgtm :+1:

File: ansible/roles/concord-demo/templates/concord.conf.j2 (hunk 1 above)
Comment (emaxerrno) [inline ansible/roles/concord-demo/templates/concord.conf.j2]: use proper versioning. 0.23.1 instead of just libmesos.so

Comment (emaxerrno) [inline ansible/roles/concord-demo/templates/concord.conf.j2]: you can probably use the same script i had which auto downloads concord

and caches it if you want. no big deal either way. 

also no jvm properties will cause it to OOM, etc.

**Comment by emaxerrno** [comment]
consider using opttions for jvm and script. use proper version and merge

LGTM after these minor changes.

File: ansible/roles/concord-demo/templates/concord.conf.j2 [hunk 2]
```diff
@@ -0,0 +1,16 @@
+description "concord"
+
+start on startup
+
+respawn
+
+env LD_LIBRARY_PATH=/usr/local/lib
+env MESOS_NATIVE_JAVA_LIBRARY=/usr/local/lib/libmesos-{{ concord_mesos_version }}.so
+exec java {% for k, v in concord_java_options.iteritems() %}-{{k}}={{v}} {% endfor %} -cp \
```
Comment (emaxerrno) [inline ansible/roles/concord-demo/templates/concord.conf.j2]: what's w/ the -X ? 

not all options are - 

also, they never really change, what's the value of making them configurable ?

File: ansible/roles/concord-demo/vars/main.yml [hunk 3]
```diff
@@ -1,2 +1,5 @@
 ---
 concord_version: 0.3.6
+concord_mesos_version: 0.23.1
+concord_java_options:
+  "DMESOS_NATIVE_JAVA_LIBRARY" : /usr/local/lib/libmesos-{{ concord_mesos_version }}.so
```
Comment (emaxerrno) [inline ansible/roles/concord-demo/vars/main.yml:11]: good naming to 23.1

File: ansible/roles/concord-demo/templates/concord.conf.j2 (hunk 2 above)
Comment (emaxerrno) [inline ansible/roles/concord-demo/templates/concord.conf.j2]: actually nvm. it doesn't matter, we aren't using non - options.

Comment (emaxerrno) [inline ansible/roles/concord-demo/templates/concord.conf.j2]: also missing -xmx xms, heapdump, etc.

**Pull request merged**